index.js

```diff
@@ -143,6 +144,11 @@ module.exports = function (content) {
     generatorOptions.dest = path.resolve(this.context, fontConfig.dest);
   }
 
+  // Spit out SCSS file to same path as CSS file to easily use mixins (scssFile must be true)
+  if (fontConfig.scssFile === true) {
+    generatorOptions.cssDest = path.resolve(this.context, fontConfig.dest, fontConfig.fontName + '.scss')
```

Comment:
@evandiamond @jeerbl 

Hey guys, After updating to the latest and just trying to set `scssFile: true` I get the following error in the webpack (^4.28.4) builds due to a `undefined` `fontConfig.dest` on this line.

```log
ERROR in ./src/assets/fonts/iconsfonts.js (./node_modules/css-loader/dist/cjs.js??ref--5-1!./node_modules/postcss-loader/src??ref--5-2!./node_modules/webfonts-loader??ref--5-3! ./src/assets/fonts/iconsfonts.js)
Module build failed (from ./node_modules/webfonts-loader/index.js):
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at assertPath (path.js:39:11)
    at Object.resolve (path.js:1085:7)
    at Object.module.exports (..../node_modules/webfonts-loader/index.js:149:37)
 @ ./src/assets/fonts/fluenticons.font.js 1:14-220
```

#### Sample config:
```js
module.exports = {
    files: ['../icons/[^_]*.svg'],
    fontName: 'mycons',
    cssTemplate: `./mycons.css.hbs`,
    baseSelector: `.mycons`,
    classPrefix: '.myicons-',
    types: ['eot', 'woff', 'woff2', 'ttf', 'svg'],
    scssFile: true,
    fileName: 'org.[fontname].[hash].[ext]'
};

```
#### I think:
- the existence of `fontConfig.dest` value must be checked before it is used. This is already done [here](https://github.com/evandiamond/webfonts-loader/blob/956e3f65ea7c1da514dfeecb84a2b32a97324827/index.js#L143-L145) in a previous code block. 
- Should the `dest` config for webfonts-generator must be set for this to work.
- Does this config need a a sass loader in the webpack rules as well?

Based on the existing doc, I am expecting a generated `scss` files in addition to the CSS files. This would be super useful from a style library perspective. However I am not sure If my understanding is correct. Could you please confirm?

Thanks

Reply:
@abhishekdev I'm not getting that error you mentioned above. It might need a check in there, but it's working with the following code. If you have a fix, I'm sure @jeerbl would be happy to accept a PR from you :)

```
// Custom Font
const fileName = 'icon-font';
const fileTemplatePath = 'icon-font-templates';

module.exports = {
    'files': [
        '../fonts/icon-font-svg/*.svg'
    ],
    'css': false,
    'scssFile': true,
    'html': true,
    'writeFiles': true,
    'fixedWidth': true,
    'embed': true,
    'fontName': fileName,
    'classPrefix': fileName + '-',
    'baseSelector': fileName,
    'types': ['eot', 'woff', 'woff2', 'ttf', 'svg'],
    'cssTemplate': '../fonts/' + fileTemplatePath + '/scss.hbs',
    'htmlTemplate': '../fonts/' + fileTemplatePath + '/html.hbs',
    'cssFontsUrl': '../fonts/' + fileName + '/',
    'dest': isHot ? '../fonts/' + fileName : '../../dist/fonts/',
    'htmlDest': '../views/font-preview.html'
};
```

Reply:
@evandiamond Thanks for sharing your working config. I see that you have a value defined for the `dest` key. With the default scenario outlined in the docs (and the tests), I would assume `webpackConfig.output.path` to be the default destination for files. The build errors on my side resolve if I set any value (even an empty string) for the `dest` key. However I still do not see any `scss` file. I would be happy to fix the behavior by correcting docs or in code....however, as of now I am not sure that I understand the expected behavior. 

**FWIW**: This error can be reproduced ([refer this branch](https://github.com/abhishekdev/webfonts-loader/tree/test-scssFile)) by updating the test case of this project to use the `scssFile` property.  The updated  test case is based on my understanding (which may be incorrect).

What I am not sure from the docs is:

1. What is the minimum configuration to create a scssFile instead?
1. Is it mandatory to set the `dest` key when the `scssFile=true`?
1. Am I missing a key assumption in the `scss.hbs` file in the config you use?
1. Would we need a scss webpack loader as well?

NOTE:  I could not generate a scssFile by appending the following 3 keys to the test case.

```js
{
  'scssFile': true,
  'css': false,
  'dest': 'dist',
}
```

Reply:
@abhishekdev is right. this 'dest' key is required but has no effect on the output.